src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String color;
+
+    public Ball(int number, String color) {
+        this.number = number;
+        this.color = color;
+    }
+
+    public Ball(Ball ball) {
+        this.number = ball.getNumber();
+        this.color = ball.getColor();
+    }
```

Comment:
it isn't very great idea to pass Ball in the constructor, let's remove this constructor from here